### PR TITLE
kubelet syncloop should stop if/when updates chan closes

### PR DESCRIPTION
**Pull request by guenter**
xref mesosphere/kubernetes-mesos#184

### CODE REVIEW

**Comment by guenter** [comment]
cc @timothysc

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -1632,7 +1632,10 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 		unsyncedPod := false
 		podSyncTypes := make(map[types.UID]metrics.SyncPodType)
 		select {
-		case u := <-updates:
+		case u, ok := <-updates:
+			if !ok {
```
Comment (yujuhong) [inline pkg/kubelet/kubelet.go]: Could you add a logging message here?

**Comment by yujuhong** [comment]
Thanks for the PR! LGTM overall with one nit.

**Comment by timothysc** [comment]
+1  @vmarmol | @dchen1107

**Comment by guenter** [comment]
@yujuhong nudge

**Comment by preillyme** [comment]
@guenter did you add the logging?

**Comment by yujuhong** [comment]
@guenter, LGTM. Thanks for the PR!

**Comment by ConnorDoyle** [comment]
@preillyme https://github.com/guenter/kubernetes/commit/3932dfd8bbdf0809fad5ec7331b7ca364d455f2a#diff-bf28da68f62a8df6e99e447c4351122dR1637

**Comment by preillyme** [comment]
@ConnorDoyle thanks for the link I seem to have missed that one.

**Pull request merged**